util/Report.cc

```diff
@@ -274,10 +306,14 @@ Report::vfileError(int /* id */,
 ////////////////////////////////////////////////////////////////
 
 void
-Report::critical(int /* id */,
+Report::critical(int id,
                  const char *fmt,
                  ...)
 {
+  // Skip suppressed messages.
+  if (suppressed_msg_ids_.find(id) != suppressed_msg_ids_.end())
+    return;
```

Comment:
You cannot ignore a critical error. This is a condition that means all hope is lost.

Reply:
Removed suppression